### ISSUE TITLE
[node e2e flake] etcd binary not found

### ISSUE THREAD

**Issue by timstclair** [kubernetes/kubernetes#29932]
From https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/29879/node-pull-build-e2e-test/16606/:

```
/var/lib/jenkins/workspace/node-pull-build-e2e-test/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e_node/e2e_node_suite_test.go:153
Unable to start node services.
etcd binary not found
/var/lib/jenkins/workspace/node-pull-build-e2e-test/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e_node/e2e_node_suite_test.go:123from junit_03.xml
```

**Comment by timstclair** [comment]
I think this is specific to the new GCI test.

**Comment by Random-Liu** [comment]
I've seen several different kinds of errors on gci.

All pods pending: https://pantheon.corp.google.com/storage/browser/kubernetes-jenkins/logs/kubelet-gce-e2e-ci/7586

Faild to prepull image: https://pantheon.corp.google.com/storage/browser/kubernetes-jenkins/logs/kubelet-gce-e2e-ci/7603

Can't find etcd: This issue.

**Comment by Random-Liu** [comment]
This should have been fixed by #30198.
Close this one.

Feel free to reopen if it happens again. :)
